suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -124,8 +134,8 @@ public Suggester(
             final boolean projectsEnabled,
             final Set<String> allowedFields,
             final int timeThreshold,
-            final int rebuildParallelismLevel
-    ) {
+            final int rebuildParallelismLevel,
+            MeterRegistry registry) {
```

Comment:
Hmm, it seems we are facing telescoping constructor anti-pattern. I'll try to think of something (builder, etc.). Feel free to ignore for now.

Reply:
okay, ignoring :-)